libs/npm/core/src/components/form/timeinput/TimeInput.tsx

```diff
@@ -5,7 +5,8 @@ import { Error } from '../../form';
 import { Show } from '../../show';
 import * as styled from './styled';
 
-export interface ITimeInputProps extends InputHTMLAttributes<HTMLInputElement> {
+export interface ITimeInputProps
```

Comment:
There are typescript issues with this when upgrading packages.  I've left it in since we don't use these properties anyway.  The MaskInput is from a dead project too...